src/libs/types/index.ts

```diff
@@ -19,6 +19,9 @@ export enum EvmChain {
   MOONBEAM = "moonbeam",
   AURORA = "aurora",
   BINANCE = "binance",
+  ARBITRUM = "arbitrum",
+  CELO = "celo",
+  KAVA = "kava",
```

Comment:
Old discussion but..
Shouldn't we use the CHAINS constant instead? 
The EvmChain should (I think) store chain id, not chain names. And chain id are environment specific. Having several EvmChain enums will be confusing.

For instance it should be `Celo` on mainnet (with first capital). But `celo` in testnet. It should be `ethereum-2` on testnet and `Ethereum` on mainnet.